src/main/kotlin/org/jetbrains/research/testspark/tools/llm/generation/TestsAssembler.kt

```diff
@@ -224,4 +223,12 @@ class TestsAssembler(
             return null
         }
     }
+
+    private fun getJUnitVersion(): JUnitVersion {
```

Comment:
Same function in TestStorageProcessingService.kt. Need to move it somewhere. Also, it can be replaced with `JUnitVersion.values().lastOrNull { it.name == settingsState.junitVersion } ?: JUnitVersion.JUnit4` , but mb it's a matter of taste